ggshield/cmd/auth/logout.py

```diff
@@ -87,10 +80,16 @@ def revoke_token(config: Config, instance_url: str) -> None:
     try:
         response = client.post(endpoint="token/revoke")
     except ConnectionError:
-        raise LogoutError(REVOKE_FAIL_MESSAGE)
+        raise click.ClickException(
+            "Could not connect to GitGuardian.\nCheck your internet connection."
```

Comment:
@martinfaucheux do we agree that this error will also be displayed when trying to run the `login` command without network as well (not only the `logout` command)?

Reply:
In fact I just realized this error can be due to 2 things
* invalid url
* no internet

How can I combine both messages? I propose
"Could not connect to GitGuardian.\nCheck the URL and your internet connection."

I'll apply the same thing for the login error.

Reply:
invalid url looks too broad no? 

eg: you want a specific error for an invalid SSO url for example no?

Reply:
I can't know if the problem is because of internet connection or invalid url. I want an message that cover both case

Reply:
@martinfaucheux then "Could not connect to GitGuardian.\nPlease check your internet connection or if the specified URL is correct."